service/runtime/handler.go

```diff
@@ -40,22 +42,24 @@ var (
 	connectionHeader     = http.CanonicalHeaderKey("connection")
 	userAgentHeader      = http.CanonicalHeaderKey("user-agent")
 
+	upvestClientID = "upvest-client-id"
```

Comment:
We have this as a constant in the platform package ready: `clientidhttp.UpvestHeader`
(Same in the test)

Reply:
This repo shouldn't have any imports from our private repos.

Reply:
Ah I see, then it's ok :)